int16 can_id
int16 data